src/modules/NtpClient.rb

```diff
@@ -153,6 +153,12 @@ def progress?
     def sync_once(server)
       log.info "Running ont time sync with #{server}"
 
+      # E.g. in inst-sys there is no chrony user. We have to
+      # create one manually before calling chronyd.
```

Comment:
I still believe it should be handle by chrony itself or installation images, so I prefer if we can wait for @wfeldt

Reply:
I've created https://github.com/openSUSE/installation-images/pull/223 for this